packages/breadcrumb/tests/Breadcrumb.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { render } from "enzyme";
+import toJson from "enzyme-to-json";
+
+import { Breadcrumb } from "../";
+
+describe("Breadcrumb", () => {
+  it("default", () => {
+    expect(
+      toJson(
+        render(
+          <Breadcrumb>
+            <span>One</span>
```

Comment:
I find it a little weird to have spans and not some custom element to render these.

Reply:
In a real application this will likely be a `Link` or as in DC/OS an image and a Link.